emcc.py

```diff
@@ -412,9 +409,14 @@ def standardize_setting_change(key, value):
     if key in shared.Settings.internal_settings:
       exit_with_error('%s is an internal setting and cannot be set from command line', key)
 
+    if key in shared.Settings.legacy_settings and key in shared.Settings.alt_names:
+      actual_key = shared.Settings.alt_names[key]
+    else:
+      actual_key = key
+
```

Comment:
`key` is used in a few places lower down, and not `actual_key` (like `if key == 'EXPORTED_FUNCTIONS'`). that seems to be a risk for bugs. can we do `key = actual_key` or not even have `actual_key`? (alternatively, doing the aliasing in the standardize function like before achieves the same)

Reply:
Ah yes.   Fixed.